src/test/javascript/spec/component/grading-system/interval-grading-system.component.spec.ts

```diff
@@ -0,0 +1,337 @@
+        lowerBoundPercentage: 40,
+        upperBoundPercentage: 65,
+        lowerBoundInclusive: true,
+        upperBoundInclusive: false,
+        isPassingGrade: true,
+    };
+    const gradeStep3: GradeStep = {
+        gradeName: 'Excellent',
+        lowerBoundPercentage: 65,
+        upperBoundPercentage: 100,
+        lowerBoundInclusive: true,
+        upperBoundInclusive: true,
+        isPassingGrade: true,
+    };
+
+    const gradeStep4: GradeStep = {
+        gradeName: 'Sticky',
+        lowerBoundPercentage: 100,
+        upperBoundPercentage: 200,
+        lowerBoundInclusive: false,
+        upperBoundInclusive: true,
+        isPassingGrade: true,
+    };
+    const gradeSteps = [gradeStep1, gradeStep2, gradeStep3, gradeStep4];
+
+    const exam = new Exam();
+    exam.maxPoints = 100;
+    const course = new Course();
+    course.maxPoints = 100;
+
+    function validateGradeStepBounds(actualGradeStepRow: any, percentageLowerBound: number, percentageUpperBound: number, maxPoints: number) {
+        expect(actualGradeStepRow.lowerBoundPercentage).toBe(percentageLowerBound);
+        expect(actualGradeStepRow.upperBoundPercentage).toBe(percentageUpperBound);
+
+        const multiplier = maxPoints / 100;
+        expect(actualGradeStepRow.lowerBoundPoints).toBe(percentageLowerBound * multiplier);
+        expect(actualGradeStepRow.upperBoundPoints).toBe(percentageUpperBound * multiplier);
+    }
+
+    beforeEach(() => {
+        TestBed.configureTestingModule({
+            imports: [ArtemisTestModule],
+            declarations: [
+                MockDirective(NgModel),
+                MockDirective(NgSelectOption),
+                IntervalGradingSystemComponent,
+                MockComponent(GradingSystemInfoModalComponent),
+                MockComponent(ModePickerComponent),
+                MockDirective(DeleteButtonDirective),
+                MockPipe(ArtemisTranslatePipe),
+            ],
+            providers: [
+                { provide: ActivatedRoute, useValue: route },
+                MockProvider(ExamManagementService),
+                { provide: CourseManagementService, useClass: MockCourseManagementService },
+            ],
+        })
+            .compileComponents()
+            .then(() => {
+                fixture = TestBed.createComponent(IntervalGradingSystemComponent);
+                comp = fixture.componentInstance;
+
+                gradingSystemService = TestBed.inject(GradingSystemService);
+                examService = TestBed.inject(ExamManagementService);
+                translateService = TestBed.inject(TranslateService);
+
+                comp.gradingScale = new GradingScale();
+                comp.gradingScale.gradeSteps = cloneDeep(gradeSteps);
+                comp.courseId = 123;
+                comp.examId = 456;
+                comp.firstPassingGrade = 'Pass';
+                translateStub = jest.spyOn(translateService, 'instant');
+            });
```

Comment:
Assigned but unused values

Reply:
Ah you are right, I usually Ctrl+F "client-style" in the Files tab but I missed those because they were in a file hidden due to "Large diffs are not rendered by default." Thanks for the catch, I'll fix these 👍

Reply:
Removed unused values.